src/analyzer.ts

```diff
@@ -402,20 +402,23 @@ export class Analyzer {
       }
     }
 
+    const files = this.#processFiles(placeholder, packageJson, syntaxInfo);
+
     if (
       wireitConfig !== undefined &&
-      command === undefined &&
       dependencies.length === 0 &&
       !dependenciesErrored &&
-      !commandError
+      command === undefined &&
+      !commandError &&
+      (files === undefined || files.values.length === 0)
```

Comment:
This PR is so elegant and minimal  I'm doing a quick clarification. Checking my understanding that you've allowed file-only scripts by making the error more permissive. And it "just works".

Nice.

Reply:
Yep!